flutter/lib/src/integrations/on_error_integration.dart

```diff
@@ -41,15 +39,7 @@ class OnErrorIntegration implements Integration<SentryFlutterOptions> {
     _defaultOnError = wrapper.onError;
 
     _integrationOnError = (Object exception, StackTrace stackTrace) {
-      _options!.logger(
-        SentryLevel.error,
-        "Uncaught Platform Error",
-        logger: 'sentry.platformError',
-        exception: exception,
-        stackTrace: stackTrace,
-      );
```

Comment:
maybe we should keep the logging if it was handled, otherwise we might swallow the error?

Reply:
Sounds reasonable to me. AS I understood, users will only get logging info if the method returns false, right?

Reply:
yup exactly

from flutter api docs:

This callback must return true if it has handled the error. Otherwise, it must return false and a fallback mechanism such as printing to stderr will be used